client/src/App.css

```diff
@@ -92,4 +116,19 @@ textarea {
 
 .type-selector {
   margin-left: 30%;
+}
+
+.userInput {
+  display: flex;
+  margin-left: 3%;
+  margin-top: 1%;
+  flex-direction: column;
+}
```

Comment:
This part is responsible for the appearance of the questions + text inputs + dropdown menu in Register.